processor/Cargo.toml

```diff
@@ -22,6 +22,8 @@ concurrent = ["std", "winter-prover/concurrent"]
 default = ["std"]
 std = ["vm-core/std", "winter-prover/std", "thiserror/std"]
 testing = ["miden-air/testing"]
+# Like `testing`, but slows down the processor speed to make it easier to debug.
+slow-testing = ["testing", "miden-air/testing"]
```

Comment:
The purpose of this feature isn't quite clear to me. It seems like we use it primarily to enable the bus debugger code. I'm assuming we do this so that not to slow down testing when we don't need to use the bus debug. If so, should we rename it to just `bus-debugger`?

Reply:
> I'm assuming we do this so that not to slow down testing when we don't need to use the bus debug.

Yes exactly. There's a significant performance drop when running the `miden-base` test suite. Although this might not be necessary if we switch to using a `BTreeMap` as you suggested - otherwise yes I agree, `bus-debugger` would be a better name